### ISSUE TITLE
Add BioConductor training materials

### ISSUE THREAD

**Issue by njall** [ElixirTeSS/TeSS_scrapers#51]
The training materials are listed in a TSV format here;
https://github.com/Bioconductor/bioconductor.org/blob/master/etc/course_descriptions.tsv

For each material you can add the title column; keyword; instructor (as author); and URL. Extract and add the first 'material' url as the link; and any subsequent URLs as associated resources.
All can be linked with the bio.tools Tool; Bioconductor:
https://bio.tools/bioconductor

Any upcoming events can be added as events and they should have a link to the added material too!

**Comment by knirirr** [comment]
What are the criteria for determining if something is a Material or an Event in that list, please? I presume that "Talk" or "Conference" would count as an event, but I'm not sure about other categories.

**Comment by knirirr** [comment]
Looking back on a Skype chat with @fbacall I see the following suggestions:

- Leave out events.
- Possibly leave out 'Conference' and 'Talk'.
- Unclear whether to leave out 'Introduction' entries.

Any comments, @njall ?

**Comment by knirirr** [comment]
I've modified this and it should be ready to go - presumably this ticket can be closed if it works on tess.oerc.

**Comment by knirirr** [comment]
Should this scraper be added to the list so it runs?

**Comment by knirirr** [comment]
Looks like this is working, and so could perhaps be deployed if you're happy, @njall .

**Comment by njall** [comment]
Thanks, made a few changes https://github.com/ElixirTeSS/TeSS_scrapers/commit/6c95896c1072478f465ac4560b6ee94edc72a94c
- I noticed the URL is to bio.tools and the logo is the ELIXIR one. I meant to add bio.tools link as an associated tools rather than the content provider URL. Fixed that; sorry for not being clearer
- I've done a horrible filter of the keywords as some are topics, some are resource types, and some are difficulty levels. It's not a sustainable solution but help seed some better data to filter over on TeSS.
